src/pydocstyle/config.py

```diff
@@ -183,6 +183,7 @@ class ConfigurationParser:
     )
     BASE_ERROR_SELECTION_OPTIONS = ('ignore', 'select', 'convention')
 
+    DEFAULT_IGNORE = {"D121", "D123"}
```

Comment:
Other tools have this kind of "we have the ability to report these codes, but their off by default" functionality. I figured this belonged to that category. Note if people are selecting `ignore`s today, this will no longer be the default.